.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,11 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Discussions
+    url: https://github.com/btcsuite/btcwallet/discussions
```

Comment:
Note that we need to enable `Discussions` here

Reply:
Enabled: https://github.com/btcsuite/btcwallet/discussions/986